CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 
 ## Unreleased
 
+- bugfix: Graphite scaler should use latest datapoint returned, not earliest  (https://github.com/kedacore/keda/pull/2365)
```

Comment:
Could you please move this to improvements section? It's a few lines below. With a message like this:

`Graphite Scaler: use the latest datapoint returned, not the earliest  (https://github.com/kedacore/keda/pull/2365)`